http/cookie.ts

```diff
@@ -291,12 +291,10 @@ export function deleteCookie(
 function parseSetCookie(value: string): Cookie | null {
   const attrs = value
     .split(";")
-    .map((attr) =>
-      attr
-        .trim()
-        .split("=")
-        .map((keyOrValue) => keyOrValue.trim())
-    );
+    .map((attr) => {
+      const [key, ...values] = attr.trim().split("=").map((keyOrValue) => keyOrValue.trim());
```

Comment:
`.map((keyOrValue) => keyOrValue.trim())`
Is ^--this part correct?

What happens if the key-value pair is like `a=foo= bar`?

Reply:
I think you are right @kt3k!